pkg/metrics/metrics.go

```diff
@@ -144,8 +144,8 @@ func getWebhooksWithLabelValues(metric prometheus.CounterVec, wt utils.WebhookTy
 		"type": wt.String(), "group": gvk.Group, "version": gvk.Version,
 		"kind": gvk.Version, "namespace": nn.Namespace, "name": nn.Name,
 	})
-	var m io_prometheus_client.Metric
-	counter.Write(&m)
+	var m prometheusclient.Metric
+	_ = counter.Write(&m)
```

Comment:
Why do we need to write the result to a variable we do not use? What is the advantage of `_ = counter.Write(&m)` over `counter.Write(&m)`?

Reply:
Oh, this just makes the IDE happy, so it won't complain about the unused return values.

Reply:
Ok. Both approaches work for me. What error do you get in your IDE? What extension is that? Mine does not complain